src/aspire/estimation/covar2d.py

```diff
@@ -633,9 +633,9 @@ def get_cwf_coeffs(self, coeffs, ctf_fb, ctf_idx, mean_coeff, covar_coeff, noise
             mean_coeff_k = ctf_fb_k.apply(mean_coeff)
 
             coeff_est_k = coeff_k - mean_coeff_k
-            coeff_est_k = sig_noise_covar_coeff.solve(coeff_est_k)
+            coeff_est_k = sig_noise_covar_coeff.solve(coeff_est_k.T)
```

Comment:
Add transpose here and remove it in line 638.

Reply:
It looks that it need to transpose back again in line 637 for apply function and the transpose in 638 is still needed. This statements are similar to those in the base class.

Reply:
This is because the `apply`/`solve` functions operate with a column-major convention. We may want to change this in the future, but the workaround right now should be to transpose the input and transpose the output for each `apply`/`solve` call to make that clear. As it is right now, sometimes `coeff_est_k` is row-major and sometimes it is column-major, which is confusing.

Reply:
Added transpose in new commit 2cd4e2d.

Reply:
I don't understand. Didn't you say that the input needed to be transposed again for `apply` on line 637?

Reply:
I went through it again yesterday. I probably put additional T for input of solve or some where. The version in the commit should be OK.

Reply:
I still don't understand. Why does the input of `solve` need to be transposed, but not the input of `apply`? Is this code being tested?

Reply:
My original changes to these lines are using the same structures in lines 306 to 308 of the base class which have the unit test for cwf coefficients. I were able to run through the experiment example.  I will add the unit test in this PR to remove the confusion instead of in different PR.

Reply:
Merged with the develop, changed to use row major for the cwf coefficients in all related statements, and added the unit test for issue #294 in the new commit df790ca.

Reply:
Ok. I'm not sure how this ran before, but the code right now looks consistent.